### DISCUSSION TITLE
how can i draw single line text,

### DISCUSSION THREAD

**Discussion by cuican6** [SixLabors/ImageSharp#2407] [Q&A]
my project,screen is 128 * 64,i want draw string like "hollo world"  in this screen

**Comment by JimBobSquarePants** [comment]
We'd love to help you with your query but you really haven't provided enough detail for us to do so.

**Comment by cuican6** [comment]
thank you sir,
i want draw string in screen, i want the font like cad's font, like this
![Uploading image.png…]()

**Reply by JimBobSquarePants** [reply]
Your image link is broken.

What do you mean by "in screen"? Do you mean a Window handle?
